hawkbit-repository/hawkbit-repository-jpa/src/main/java/org/eclipse/hawkbit/repository/jpa/model/JpaDistributionSet.java

```diff
@@ -347,14 +347,14 @@ public void fireUpdateEvent(final DescriptorEvent descriptorEvent) {
                 new DistributionSetUpdateEvent(this, EventPublisherHolder.getInstance().getApplicationId()));
 
         if (isSoftDeleted(descriptorEvent)) {
-            publishEventWithEventPublisher(new DistributionSetDeletedEvent(getTenant(), getId(),
+            publishEventWithEventPublisher(new DistributionSetDeletedEvent(getTenant(), getId(), getClass().getName(),
```

Comment:
Have a look here: http://stackoverflow.com/a/18684976 this is explaining the background of my question very well. So a fixed type `JpaDistributionSet.class.getName()` may be better here?